spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -313,6 +313,27 @@ deployment request prefixed by the `app.<name of application>`.  For example
 
 will deploy the time source with 2048MB of memory, while the log sink will use the default 1024MB.
 
+=== Using Spring Cloud Config Server
+If using Spring Cloud Config Server as a Cloud Foundry service, the easiest way to externalize the above configuration
+and consume it from the Data Flow server is to use the `spring-cloud-services-starter-config-client` dependency.
+As this support is specific to Pivotal Cloud Foundry, it is not included by default. But building a Data Flow server
+that embeds it is as simple as adding a dependency similar to
+
+[source, xml]
+```
+<dependency>
+    <groupId>io.pivotal.spring.cloud</groupId>
+    <artifactId>spring-cloud-services-starter-config-client</artifactId>
+    <version>1.1.0.RELEASE</version>
+    <scope>runtime</scope>
+</dependency>
+```
+
+to your own version of the
+https://github.com/spring-cloud/spring-cloud-dataflow-server-cloudfoundry/tree/master/spring-cloud-dataflow-server-cloudfoundry-docs[Data Flow server],
```

Comment:
Link should point to: https://github.com/spring-cloud/spring-cloud-dataflow-server-cloudfoundry/tree/master/spring-cloud-dataflow-server-cloudfoundry